pkger/service_logging.go

```diff
@@ -148,7 +148,8 @@ func (s *loggingMW) Export(ctx context.Context, opts ...ExportOptFn) (template *
 			s.logger.Error("failed to export template", zap.Error(err), dur)
 			return
 		}
-		s.logger.Info("failed to export template", append(s.summaryLogFields(template.Summary()), dur)...)
+		// todo: should these be Debug logs?
```

Comment:
@russorat are you the man to answer this?

Reply:
Based on what i see in other parts of the app, i would think these "failed to" logs should be debug.

Reply:
the `failed to` seemed to be copypasta. the logs in question are the success logs

https://github.com/influxdata/influxdb/pull/19457/files/53b325c7d0d16ce11723083cd3a0243e481881ce#diff-11efed3cf4ce3c7284b325143fb8b70dR180-R181
```go
s.logger.Info("template dry run successful", fields...)
// ...
s.logger.Info("template apply successful", fields...)
```

Reply:
Ah, if they are success logs, i would think they should be Info